helm/charts/aleph/templates/ingest-file.yaml

```
@@ -103,13 +104,35 @@ spec:
   maxReplicas: {{.Values.ingestfile.hpa.maxReplicas}}
   metrics: {{- toYaml .Values.ingestfile.hpa.scalingMetrics | nindent 4}}
 ---
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
+{{ if .Values.global.prometheus.enabled }}
+apiVersion: v1
+kind: Service
 metadata:
-  name: {{ .Values.global.namingPrefix }}-ingest-file-policy
+  name: {{ .Values.global.namingPrefix }}-ingest-file
+  labels:
+    app: {{ .Values.global.namingPrefix }}-ingest-file
 spec:
-  podSelector:
+  selector:
+    app: {{ .Values.global.namingPrefix }}-ingest-file
+  ports:
+    - port: 9100
+      targetPort: 9100
+      protocol: "TCP"
+      name: metrics
+{{ end }}
+---
+{{ if .Values.global.serviceMonitor.enabled }}
```

```suggestion
{{ if and .Values.exporter.enabled .Values.global.serviceMonitor.enabled }}
```

This should be consistent across all templates.